lib/feature.pm

```diff
@@ -271,8 +271,9 @@ regardless of what feature declarations are in scope.
 
 The 'postderef_qq' feature extends the applicability of L<postfix
 dereference syntax|perlref/Postfix Dereference Syntax> so that postfix array
-and scalar dereference are available in double-quotish interpolations. For
-example, it makes the following two statements equivalent:
+and scalar dereference and postfix array length access are available in
+double-quotish interpolations. For example, it makes the following two
```

Comment:
The "X and Y and Z" syntax is sub-optimal English style.  You probably want:
```
dereference syntax|perlref/Postfix Dereference Syntax> so that postfix array,
scalar dereference and postfix array length access are available in
double-quotish interpolations.
```

Reply:
I was going for it to parse as:
```
  (postfix array and scalar dereference) and (postfix array length access)
```
Perhaps some other way of rearranging it might work:
```
  postfix array/scalar dereference and postfix array length access
```
?

Reply:
Or use separate sentences like I did in `perlref.pod`.

Reply:
I'll probably go with

> postfix array dereference, postfix scalar dereference, and postfix array length access

as this is the least ambiguous despite the extra length.